lib/classes/Swift/Transport/StreamBuffer.php

```diff
@@ -264,7 +264,7 @@ private function _establishSocketConnection()
             $options = array_merge($options, $this->_params['stream_context_options']);
         }
         $streamContext = stream_context_create($options);
-        $this->_stream = @stream_socket_client($host.':'.$this->_params['port'], $errno, $errstr, $timeout, STREAM_CLIENT_CONNECT, $streamContext);
+        $this->_stream = stream_socket_client($host.':'.$this->_params['port'], $errno, $errstr, $timeout, STREAM_CLIENT_CONNECT, $streamContext);
```

Comment:
`$errno, $errstr` should contain the relevant error, and they are part of the exception message. So, I suppose you found a case where it was not the case?